Source/WebCore/platform/graphics/ca/cocoa/PlatformCALayerCocoa.mm

```diff
@@ -930,7 +930,7 @@ - (void)setOwner:(WebCore::PlatformCALayer*)owner
         return false;
     
     for (unsigned i = 0; i < filters.size(); ++i) {
-        const FilterOperation* filterOperation = filters.at(i);
+        RefPtr filterOperation = filters.at(i);
         switch (filterOperation->type()) {
```

Comment:
If type is trivial, you can also just get rid of the local variable and just call `switch (filters.at(i)->type()) {` here.